src/Executor/ExecutionResult.php

```diff
@@ -8,7 +8,6 @@
 use GraphQL\Error\Error;
 use GraphQL\Error\FormattedError;
 use JsonSerializable;
-use ReturnTypeWillChange;
```

Comment:
this will provoke warning in php, unrelated to phpstan

Reply:
This was just one idea I had, feel free to push another attempt to this PR.